### DISCUSSION TITLE
Images are Black with white text, how can i invert?

### DISCUSSION THREAD

**Discussion by ilcaa72** [sumatrapdfreader/sumatrapdf#3643] [General]
my **_images_** are printing with a black background and white text, i want to invert them.  white background, black text, again only for images

where is setting for this?  thank you!

**Comment by GitHubRulesOK** [comment]
SumatraPDF should only offer inversion for viewing say blueprints or negatives, but will/can not reverse them for printing.

SOME may be reversible in MuPDF MuTools and others you may need other approaches, 
but basically it is in effect an edit you need to negate images, and many apps such as IrfanView can be scripted to modify files including PDF.

While viewing such a file it may be possible to send the page range to annother app to reverse (edit) colours then use SumatraPDF to send the edited copy to the print system.
